signbank/settings/development.py

```diff
@@ -4,6 +4,22 @@
 
 from signbank.settings.base import *
 
+# Needed for collectstatic
+try:
```

Comment:
issue: I'm confused by this - our `base.py` settings use `os.environ.get` without a default value to fetch the `DATABASE_URL`, so it should error if you don't provide that and I would expect `collectstatic` to load that settings file.

Is it possible this is an issue with your local setup?

Reply:
(sorry this was meant to be part of my review, but apparently I didn't post this comment before submitting the review)

Reply:
No, happens on a vanilla setup so it's not me.
Maybe I'll just get it to define DATABASE_URL guaranteed. Depends how much more time you want me throw at it?

Reply:
Without this code, 4.1 and higher will crash on `collectstatic` with a DATABASES incorrectly-formatted error.
I'll see if there's a cleaner way to make sure it's got DATABASE_URL always everywhere.

Reply:
Ok, NOW it wants to work just fine.
Nooooooooooo idea.  I'll remove this code block.
🤷🏻‍♂️ 🤷🏻 🤷🏻‍♀️

Reply:
> Without this code, 4.1 and higher will crash on collectstatic with a DATABASES incorrectly-formatted error.

That error sounds different to what you originally said - combined with the fact that we're always expecting `DATABASE_URL` and that we use [`dj-database-url`](https://github.com/jazzband/dj-database-url), I would guess the issue is that the version of `dj-database-url` we're using is not compatible with this new version of Django.

Previously, we were using v0.5.0 and the latest version is v2.2.0 so sounds like there's been some decent amount of change in that library

Reply:
> Ok, NOW it wants to work just fine.

probably because of version confusion 🙂

Reply:
Done.

Reply:
dj-datbase-url was my first port of call, but it's not getting called when that error manifests.
It's obviously some obscure corner-case and if it turns up again we know where to start digging now.